src/tools/wasm-reduce.cpp

```diff
@@ -104,12 +217,12 @@ ProgramResult expected;
 static std::unordered_set<Name> functionsWeTriedToRemove;
 
 struct Reducer : public WalkerPass<PostWalker<Reducer, UnifiedExpressionVisitor<Reducer>>> {
-  std::string command, test, working;
+  std::string command, test, working, binaryenBins;
```

Comment:
I'm confused what does the `s` indicate in the name? is it plural?

How about `binaryenBinDir`?